Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageContentView.swift

```diff
@@ -147,6 +147,9 @@ open class _ChatMessageContentView<ExtraData: ExtraDataTypes>: _View, ThemeProvi
     /// The container which holds `onlyVisibleForYouIconImageView` and `onlyVisibleForYouLabel`
     public private(set) var onlyVisibleForYouContainer: ContainerStackView?
 
+    /// Constraint between bubble and reactions.
+    public private(set) var bubbleToReactionsConstraint: NSLayoutConstraint?
+    
```

Comment:
What was the reason for this?

Reply:
I needed to turn off the constraint to layout the message content view _without_ reactions. Let me know if there is more elegant way how to achieve this. I have pondered using `func layout(options: ChatMessageLayoutOptions)` but AFAICT this method can be called only once?